### ISSUE TITLE
Final Report Peer Review by ts568

### ISSUE THREAD

**Issue by tiffanyshih** [vanshikabansal/ORIE-4741#11]
The goal of this project is clear, the group hopes to use datasets from all Hong Kong Jockey Club horse races from 2014 to 2017 to develop a model that may predict the finishing position of a certain horse at the end of a race. At the outset, the group selects certain features that they believe to be impactful in determining a horse's race position and tries to avoid any dependency on specific horses so that the model is flexible enough to be adapted to new data.

Some aspects I enjoyed about the report:
1. The group completed a very comprehensive analysis of their data. Though the data sets were very involved, the team was able to discern important features for their analysis at the outset and provides detailed analysis about which features they chose to keep and why.
2. The team provides clear transitions from model to model, explaining why they chose to fit a certain model next in their analysis. 
3. The team utilizes various data visualizations as well as tables to present their results, which greatly helps the audience understand the analysis.
4. The group provides a strong conclusion on why they are not confident in using solely their best fit model to make predictions as well as what they believe could help in terms of next steps in producing a better model.

Some things I noticed for improvement:
1. The group uses a training set containing 80% of the data and a test set containing 20% of their data to measure their errors and determine the best model. It may have been a more robust analysis if the team instead split their data into training, validation, and test sets, so that they could fit all their models on the training set, then calculate the respective errors on the validation set in order to select the one with best fit to measure test set error.
2. Since the data set seemed pretty complex, I believe the report would have benefitted from some additional clarifications. For example, what each "race class" represented. 
3. There are small grammatical errors throughout the report that make it a bit difficult to read. The report would have benefitted from some additional proofreading.
